dxr/plugins/rust/tests/__init__.py

```diff
@@ -0,0 +1,15 @@
+from dxr.testing import SingleFileTestCase
+
+class RustSingleFileTestCase(SingleFileTestCase):
+    """Test case suited to testing the Rust plugin."""
+    source_filename = 'mod.rs'
+
+    @classmethod
+    def config_input(cls, config_dir_path):
+        config = super(RustSingleFileTestCase, cls).config_input(config_dir_path)
+
+        config['DXR']['enabled_plugins'] = 'pygmentize rust'
+        config['DXR']['disabled_plugins'] = 'clang'
+        config['code']['build_command'] = '$RUSTC mod.rs'
```

Comment:
If you can't think of anything (and nothing comes to my mind), we can just rig it so $RUSTC *is* defined on the vagrant box. There's precedent for appending stuff to .bashrc in vagrant_provision.sh.